src/toucan2/jdbc/mysql_mariadb.clj

```diff
@@ -107,7 +107,7 @@
         _              (log/debugf "update-returning-pks workaround: doing SELECT with conditions %s"
                                    conditions-map)
         parsed-args    (update pipeline/*parsed-args* :kv-args merge conditions-map)
-        select-rf      (pipeline/with-init conj [])
+        select-rf      (pipeline/conj!-with-init [])
```

Comment:
I'd rather call it `conj-with-init!` or something -- having it end in an `!` makes it clearer that this is a thread-unsafe/STM-unsafe function and we have some custom linters in MB core that actually do smart things with symbols named this way